src/main/java/dev/emi/trinkets/api/Trinket.java

```diff
@@ -103,26 +103,43 @@ default RegistryEntry<SoundEvent> getEquipSound(ItemStack stack, SlotReference s
 		return stack.getItem() instanceof Equipment eq ? eq.getEquipSound() : null;
 	}
 
+	//internalizes getTrinket and slotIdentifier, both which typically are generated just before the modifiers call anyway
+	static Multimap<RegistryEntry<EntityAttribute>, EntityAttributeModifier> trinketModifiers(ItemStack stack, SlotReference slot, LivingEntity entity){
```

Comment:
I don't really understand why these exist. You cannot overload static methods. Additionally, the styling and documentation is inadequate. I feel like this should remain baked in behavior of the bottom level getModifiers

Reply:
If visibility is the concern, I can move them elsewhere. Probably best you tell we where so I'm not just hunting for the proper spot myself

Reply:
moved per conversation above/below